HTML/launches.html

```diff
@@ -6,10 +6,34 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css" integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
   <title>launches</title>
-      <link href="./CSS/launches.css" rel="stylesheet" type="text/css" />
+  
+      <link href="/CSS/launches.css" rel="stylesheet" type="text/css" />
```

Comment:
wrong path to CSS file, you have to get out of the file first and then out of the folder

```suggestion
  
      <link rel="stylesheet" href="../CSS/launches.css">
```